tests/tests/integration_tests.rs

```diff
@@ -253,7 +259,7 @@ async fn test_int8(subgraph: Subgraph) -> anyhow::Result<()> {
 * the `cases` variable in `integration_tests`.
 */
 
-async fn test_timestamp(subgraph: Subgraph) -> anyhow::Result<()> {
+async fn test_timestamp(subgraph: Subgraph, _ctx: TestContext) -> anyhow::Result<()> {
```

Comment:
If Testcontext is passed to all of these then it would probably be easier to just pass move the Subgraph argument to the Context struct and then all is needed is the Context

Reply:
Well noted, just changed